docs/docs/examples/guides/04-object-detection-example.md

```diff
@@ -0,0 +1,745 @@
+  },
+  "registration": "component_utils::YoloToMarker",
+  "inherits": "modulo_components::LifecycleComponent",
+  "inputs": [
+    {
+      "display_name": "Json Input",
+      "description": "Bounding boxes from YOLO model",
+      "signal_name": "json_input",
+      "signal_type": "string"
+    }
+  ],
+  "outputs": [
+    {
+      "display_name": "Pose Command",
+      "description": "The pose command",
+      "signal_name": "marker_pose",
+      "signal_type": "cartesian_state"
+    }
+  ],
+  "parameters": [
+    {
+      "display_name": "Object to Track",
+      "description": "The name of the object to track, matching a class from the YOLO model.",
+      "parameter_name": "to_find",
+      "parameter_type": "string",
+      "default_value": "person"
+    },
+    {
+      "display_name": "Field of View (FoV)",
+      "description": "Camera field of view in degrees [horizontal, vertical]",
+      "parameter_name": "fov",
+      "parameter_type": "double_array",
+      "default_value": [69.0, 42.0]
+    },
+    {
+      "display_name": "Image Size",
+      "description": "Image resolution [width, height] in pixels",
+      "parameter_name": "image_size",
+      "parameter_type": "double_array",
+      "default_value": [480, 840]
+    },
+    {
+      "display_name": "Object Distance",
+      "description": "Assumed distance from the camera to the object in meters",
+      "parameter_name": "object_distance",
+      "parameter_type": "double",
+      "default_value": 0.6,
+      "dynamic": true
+    }
+  ]
+}
+```
+
+</details>
+
+Enter the component folder in terminal and run
+
+```bash 
+docker build -f aica-package.toml -t objectdetection .
+```
+
+Next, configure AICA Studio and add **objectdetection** under Custom Packages. After launching, you should see
+**Component Utils** under **Add Component** and be able to add *YOLO to Marker*.
+
+### Application Setup
+
+Connect the `Bounding Boxes` output of *YOLO Executor* to `JSON Input` on *YOLO to Marker* then configure it with
+**auto-configure** and **auto-activate**. You should also set the parameters based on your camera and what you are filming.
+Set `rate` to match that of *YOLO Executor*. Set the `Object to Track` parameter to "person" to track yourself in the frame. With the application running the logs should show the 3D position in the camera_frame.
+
+#### Creating a Frame and Converting it to a Signal
+
+Add in a *Hardware Interface* if it is not already there, and run the application. When it is running, select _3D Viz_ at
+the top right. Record a frame `tool0` and name it `camera_frame`, this is the position of the camera. It can be moved by
```

Comment:
this is only valid for the generic six-axis robot arm. That's ok, but we should mention 1) which robot to choose, 2) that if another robot is chosen then this should be adapted (perhaps a :::note)